CRM/Member/BAO/Membership.php

```diff
@@ -2241,7 +2241,7 @@ public static function updateAllMembershipStatus() {
     // Tests for this function are in api_v3_JobTest. Please add tests for all updates.
 
     $updateCount = $processCount = self::updateDeceasedMembersStatuses();
-    $allStatus = CRM_Member_PseudoConstant::membershipStatus();
+    $allStatus = CRM_Member_BAO_Membership::buildOptions('status_id', 'get');
```

Comment:
So looking at https://docs.civicrm.org/dev/en/latest/framework/pseudoconstant/

![Screenshot 2019-03-18 14 51 36](https://user-images.githubusercontent.com/336308/54502054-99bc9480-498d-11e9-8db0-d9a8587ae191.png)

I see a risk - if someone relabels expired this will not pick it up - I think we should use 'validate' rather than 'get' as the context & in the unit test we should set the name & label to not be the same to test that.

(Note that there are multiple other places on the BAO where the label is checked rather than the name - I think fixing those is out of scope but perhaps it would be good to add todos to them)